common.sh

```diff
@@ -216,6 +216,7 @@ create_instance()
                     --network-interface ${network_interface}",\"SubnetId\":\"${subnet}\"}]" \
                     --count ${NODES}:${NODES} \
                     --query "Instances[*].InstanceId" \
+                    --user-data file://${JENKINS_HOME}/user_data_script.sh \
```

Comment:
Just want to double check is it guaranteed that JENKINS_HOME is defined?

Reply:
I updated it, I checked and there are multiple consumers of create instances, so I changed it to an environment variable argument.